chain/stmgr/stmgr.go

```diff
@@ -40,8 +42,7 @@ import (
 const LookbackNoLimit = api.LookbackNoLimit
 const ReceiptAmtBitwidth = 3
 
-const execTraceCacheSize = 16
-
+var defaultExecTraceCacheSize = 16
```

Comment:
Keep the old name? Because if the envvar is set, we set this field, which means it's no longer a default size?